app/src/main/java/com/funcheap/funmapsf/commons/utils/DateCostFormatter.java

```diff
@@ -62,8 +57,7 @@ public static String formatDate(String date)  {
     public static String formatDatefromDate(String date)  {
         String retDate = date;
         try {
-            SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd hh:mm:ss", Locale.ENGLISH);
```

Comment:
I had to change this from `hh` to `HH` because retrieved times are in 0-23, not 0-12. It caused 12:30 to be parsed as 12:30AM.